Sources/Mustache/ContentType.swift

```diff
@@ -50,8 +50,14 @@ public enum MustacheContentTypes {
         self.types[name] = contentType
     }
 
-    static var types: [String: MustacheContentType] = [
+    private static let _types: [String: MustacheContentType] = [
         "HTML": HTMLContentType(),
         "TEXT": TextContentType(),
     ]
+
+    #if compiler(>=6)
+    nonisolated(unsafe) static var types: [String: MustacheContentType] = _types
```

Comment:
This still needs to be protected by a lock

Reply:
56644a4 Sorry, a lock was applied in Swift 6 mode without changing the API.

Reply:
Ah I see. I would've applied the lock in a computed property here. But that's fine too.

Reply:
```swift
    nonisolated(unsafe) static var types: [String: MustacheContentType] {
        get {
            lock.withLock { _type }
        }
        set {
            lock.withLock { self.types[name] = newValue }
        }
    }
```

Is this what you mean?